assets/scripts/store/actions/street.js

```diff
@@ -52,6 +53,13 @@ export function addLocation (location) {
   }
 }
 
+export function clearLocation () {
+  return {
+    type: CLEAR_LOCATION,
+    defaultName: 'Unnamed St'
```

Comment:
Let's actually pull in the locale-based string for the default name.

- Import the `t` function from `scripts/app/locale`
- Get the locale based string by calling `t('street.default-name', 'Unnamed St')` (the first argument is a reference to the key in one of the translation files, e.g. `assets/locales/en/main.json` , and the second is a fallback string in case the translation file couldn't be read -- it also doubles as "documentation" of what the string should say without needing to look up the translation file.)